ts/Dashboard/JSON.d.ts

```diff
@@ -0,0 +1,62 @@
+/* *
+ *
+ *  (c) 2020-2022 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Sophie Bremer
+ *
+ * */
+
+/* *
+ *
+ *  Declarations
+ *
+ * */
+
+export namespace JSON {
+
+    /**
+     * Type structor of arrays as it is supported in JSON.
+     */
+    export interface Array<T extends (Primitive|Type)=(Primitive|Type)> extends globalThis.Array<T> {
+        [index: number]: T;
+    }
+
+    /**
+     * Class API for JSON.stringify.
+     */
+    export interface Builder {
+        toJSON(): Type;
+    }
+
+    /**
+     * Type structure of a record object as it is supported in JSON.
+     */
+    export interface Object<T extends (Primitive|Type)=(Primitive|Type)> {
+        [key: string]: T;
+    }
+
+    /**
+     * All primitive types, that are supported in JSON.
+     */
+    export type Primitive = (boolean|number|string|null|undefined);
```

Comment:
How about the object?

Reply:
What about it? :D I copied whole JSON.d.ts from /Core to /Dashboard as described in the task.